### PR TITLE
[CARBONDATA-2738]Update documentation for Complex datatype

### PR DESCRIPTION
**Pull request by Indhumathi27**
For PR-2501, Update documentation for Complex datatype

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [x] Document update required?
         Added
 - [ ] Testing done
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/5817/

**Comment by Indhumathi27** [comment]
Retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/7171/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/5947/

**Comment by sgururajshetty** [comment]
LGTM

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -290,6 +292,7 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
      ```
      ALTER TABLE carbon ADD COLUMNS (a1 INT, b1 STRING) TBLPROPERTIES('DEFAULT.VALUE.a1'='10')
      ```
+      NOTE: Add Complex datatype columns is not supported.
 
    - **DROP COLUMNS**
```
Comment (ajantha-bhat) [inline docs/data-management-on-carbondata.md]: complex child columns drop is not supported. Update this point.

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/7222/

**Comment by Indhumathi27** [comment]
Retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/7226/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/5998/

**Comment by ajantha-bhat** [comment]
LGTM

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/5882/

**Comment by kunal642** [comment]
LGTM

**Pull request closed without merging**